rest-server/src/controller/user_controller.cpp

```diff
@@ -67,7 +67,24 @@ void UserController::HandlePost(http_request message) {
     }
 }
 void UserController::HandleDelete(http_request message) {
-    message.reply(status_codes::NotImplemented, ResponseNotImpl(methods::DEL));
+    try {
+        auto replyStatus = status_codes::OK;
+        const json::value message_body = message.extract_json().get();
```

Comment:
I know this is just sample code, so I'm not sure it's necessary for you to change this, but if you look at the REST API spec, DELETE requests should specify the id by the URI, not in JSON. For example, a DELETE should merely have to be sent to ```http://localhost:8080/v1/cadg/api/users/12 ``` in order for the server to delete user 12's data.
API Spec: https://docs.google.com/document/d/1QO4ZuVg5PgG6w8XZGGNnwgVcuMGLhKH7uK5_4ngXvk0/edit#heading=h.91mae4u93k

Reply:
Oh man you are totally correct. I will get that changed!